### PR TITLE
Update to the latest versions (1.0.2 release)

### PR DESCRIPTION
**Pull request by chapulina**
Periodic upgrade to make sure users are getting the latest and greatest

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -18,19 +18,19 @@ ign_configure_project()
...
-ign_find_package(ignition-physics2 REQUIRED VERSION 2.3)
-ign_find_package(ignition-plugin1 REQUIRED)
-ign_find_package(ignition-rendering3 REQUIRED VERSION 3.1)
-ign_find_package(ignition-sensors3 REQUIRED)
-ign_find_package(ignition-tools REQUIRED)
-ign_find_package(ignition-transport8 REQUIRED)
-ign_find_package(sdformat9 REQUIRED VERSION 9.2.0)
+ign_find_package(ignition-common3 REQUIRED VERSION 3.14)
+ign_find_package(ignition-fuel_tools4 REQUIRED VERSION 4.4)
+ign_find_package(ignition-gazebo3 REQUIRED VERSION 3.9)
```
Comment (iche033) [inline CMakeLists.txt]: should this be the latest version, 3.11?

Comment (jennuine) [inline CMakeLists.txt]: Isn't `3.11.0` the latest release?

```suggestion
ign_find_package(ignition-gazebo3 REQUIRED VERSION 3.11)
```

Comment (chapulina) [reply CMakeLists.txt]: Good point, I updated these based on the dashboard, and the latest released version is 3.9:

http://packages.osrfoundation.org/gazebo/ubuntu-stable/pool/main/i/ignition-gazebo3/

image

Comment (jennuine) [reply CMakeLists.txt]: oops, @iche033 beat me to it

Comment (chapulina) [reply CMakeLists.txt]: I just saw this too, see my response to Ian's comment. It looks like our Gazebo 3 releases are very broken

Comment (chapulina) [reply CMakeLists.txt]: 3.11 is now fixed for Bionic and Focal amd64, so I bumped it here

Comment (chapulina) [reply CMakeLists.txt]: image

**Review by jennuine — APPROVED** [review]

**Pull request merged**